demo/config.yaml

```diff
@@ -33,3 +33,11 @@ lcr-modules:
             # if using manta output, use vcf file in the 99-outputs subdirectory and ensure manta version corresponds to the loaded module
             candidate_small_indels: "results/manta-2.2/99-outputs/vcf/{seq_type}--{genome_build}/candidateSmallIndels/{tumour_id}--{normal_id}--{pair_status}.candidateSmallIndels.vcf"
 
+    controlfreec:
+        inputs:
+            sample_bam: "data/{sample_id}.bam"
+            sample_bai: "data/{sample_id}.bam.bai"
+        logs:
+            controlfreec: "logs/"
```

Comment:
Doesn't LCR modules have a place for logs that is tool-specific (in the output directories)?

Reply:
Removed